From c3ed8088e4812d4f58bf9f6124106c65be22faef Mon Sep 17 00:00:00 2001
From: Andreas Jaeger <aj@suse.de>
Date: Mon, 29 Apr 2013 21:15:44 +0200
Subject: [PATCH 4/4] BZ#15084: Apparent typos in strings in res_debug.c

	[BZ #15084]
	* resolv/res_debug.c (p_option): Fix strings for RES_USEBSTRING
	and RES_USEVC.
---
 ChangeLog          | 4 ++++
 NEWS               | 8 ++++----
 resolv/res_debug.c | 4 ++--
 3 files changed, 10 insertions(+), 6 deletions(-)

diff --git a/resolv/res_debug.c b/resolv/res_debug.c
index c1d8c02..7843439 100644
--- a/resolv/res_debug.c
+++ b/resolv/res_debug.c
@@ -573,7 +573,7 @@ p_option(u_long option) {
 	case RES_INIT:		return "init";
 	case RES_DEBUG:		return "debug";
 	case RES_AAONLY:	return "aaonly(unimpl)";
-	case RES_USEVC:		return "usevc";
+	case RES_USEVC:		return "use-vc";
 	case RES_PRIMARY:	return "primry(unimpl)";
 	case RES_IGNTC:		return "igntc";
 	case RES_RECURSE:	return "recurs";
@@ -588,7 +588,7 @@ p_option(u_long option) {
 	case RES_NOCHECKNAME:	return "no-check-names(unimpl)";
 	case RES_KEEPTSIG:	return "keeptsig(unimpl)";
 	case RES_BLAST:		return "blast";
-	case RES_USEBSTRING:	return "ip6-bytstring";
+	case RES_USEBSTRING:	return "ip6-bytestring";
 	case RES_NOIP6DOTINT:	return "no-ip6-dotint";
 	case RES_USE_EDNS0:	return "edns0";
 	case RES_SNGLKUP:	return "single-request";
-- 
1.8.2.3

